src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/run.py

```diff
@@ -67,25 +71,61 @@ def acr_run(cmd,  # pylint: disable=too-many-locals
         source_location = check_remote_source_code(source_location)
         logger.warning("Sending context to registry: %s...", registry_name)
 
+    yaml_template = ""
```

Comment:
I have some comments in tas.py which also apply here.
1. timeout settings should be moved into `cmd_value`.
2. block if users pass file and cmd_value at the same time.

Reply:
Are you assuming timeout can be taken in differently? For --cmd, the timeout should be read from  --timeout while for piped definition, the timeout should be read from the stream. If timeouts are not used in those ways by customers, they will be skipped.